### PR TITLE
Modified MergeDataTableCommand and Fixed issue in ExcelGetRangeCommandASDT

### PR DESCRIPTION
**Pull request by ob-ab**
Added MergeDataTableCommand

Modified MergeDataTableCommand:
- Used [XMLAttributes] instead of Explicit Creation of Combobox for MissingSchemaAction Options

Issue Fixing in ExcelGetRangeCommandASDT:
- Command was breaking when reading an excel file with First Column having Empty/Null Values

### CODE REVIEW

File: taskt/Core/Automation/Commands/MergeDataTableCommand.cs [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            /* ------------Before Merge Operation, following conditions must be checked---------------
+               
+            1. None of the (Source, Destination) DataTable Variables is null            -->     (Null Check)
+            2. Data Type of both (Source, Destination) Variables must be DataTable      -->     (Data Type Check)
+            3. Source and Destination DataTable Varibales must not be the same          -->     (Same Variable Check)
+
+             */
+
+            var engine = (AutomationEngineInstance)sender;
+            var action = v_MissingSchemaAction.ConvertToUserVariable(sender);
```
Comment (openbots-ff) [inline taskt/Core/Automation/Commands/MergeDataTableCommand.cs]: This conversion is unnecessary given than you're never actually looking up an action in the engine's variable list. It always comes from the drop down options.

File: taskt/Core/Automation/Commands/MergeDataTableCommand.cs [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        [Attributes.PropertyAttributes.PropertyUISelectionOption("Error")]
+        [Attributes.PropertyAttributes.PropertyUISelectionOption("Ignore")]
+        [Attributes.PropertyAttributes.InputSpecification("Select any Missing Schema Action")]
+        [Attributes.PropertyAttributes.SampleUsage("**mySchemaAction**, **{mySchemaAction}**")]
+        [Attributes.PropertyAttributes.Remarks("")]
+        public string v_MissingSchemaAction { get; set; }
+
+        [XmlIgnore]
+        [NonSerialized]
+        public ComboBox SchemaActionComboBox;
```
Comment (openbots-ff) [inline taskt/Core/Automation/Commands/MergeDataTableCommand.cs]: please remove this combobox

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: taskt/Core/Automation/Commands/MergeDataTableCommand.cs (hunk 1 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/MergeDataTableCommand.cs]: Right.
Actually, It was overlooked.

Removed it.

File: taskt/Core/Automation/Commands/MergeDataTableCommand.cs (hunk 2 above)
Comment (ob-ab) [reply taskt/Core/Automation/Commands/MergeDataTableCommand.cs]: Removed

**Review by CameronHerwig — APPROVED** [review]
Looks good to me after the requested changes!

**Review by melugi — APPROVED** [review]
Looks good to me as well.

**Review by openbots-ff — APPROVED** [review]

**Pull request merged**